packages/plugin-kit/src/config-comment-parser.js

```diff
@@ -9,7 +9,6 @@
 // Imports
 //------------------------------------------------------------------------------
 
-// @ts-ignore -- don't feel like fighting with TypeScript right now
```

Comment:
[Question] I don't get any type errors in my editor or from `npm run build` on `main` after removing this comment directive. Is there another command I'm missing?

Reply:
Leftover comment from before I figured out how to get it to work. :smile: